.github/workflows/uberjar.yml

```diff
@@ -139,6 +170,15 @@ jobs:
       if: ${{ !(startsWith(github.ref_name,'master') || startsWith(github.ref_name,'backport')) && matrix.edition == 'ee' }}
       run: docker tag localhost:5000/metabase-dev:${{ steps.extract_branch.outputs.branch }}-ee metabase/metabase-dev:${{ steps.extract_branch.outputs.branch }} && docker push metabase/metabase-dev:${{ steps.extract_branch.outputs.branch }}
 
+      # Push experimental ubuntu image only for versions based on a master
+    - name: Retag and push ubuntu-based images if master (ee)
+      if: ${{ (github.ref_name == 'master') && matrix.edition == 'ee' }}
+      run: docker tag localhost:5000/metabase-dev:${{ steps.extract_branch.outputs.branch }}-ee-ubuntu metabase/metabase-enterprise-head-ubuntu:latest && docker push metabase/metabase-enterprise-head-ubuntu:latest
```

Comment:
we need to create this repos right?

Reply:
That is not a separate repo, that is just another one tag. The repo is the same - metabase, but the tag has an appendix "ubuntu" now.